src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -190,20 +192,28 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
     this.uploadedDocument.get(WriteDocumentFieldComponent.DOCUMENT_URL).setValue(url);
     this.uploadedDocument.get(WriteDocumentFieldComponent.DOCUMENT_BINARY_URL).setValue(binaryUrl);
     this.uploadedDocument.get(WriteDocumentFieldComponent.DOCUMENT_FILENAME).setValue(filename);
+    const value = {
```

Comment:
let's introduce a private method that gets as parameters `url`, `binaryUrl` and `document_filename` and creates the object

Reply:
OK